common/metrics/defs.go

```diff
@@ -53,6 +53,10 @@ type (
 	ServiceIdx int
 )
 
+func (s scopeDefinition) GetOperationString() string {
+	return s.operation
+}
+
```

Comment:
could remove this - it's mostly just prep-work since almost anything leaving the scope-index ints will need it.